mz_bokeh_package/components/plot_settings.py

```diff
@@ -759,7 +781,7 @@ def _set_colors_by_field(self, colors: List[str]):
             colors (List[str]): A list of colors that were chosen by the user in the settings modal.
         """  # noqa: E501
 
-        renderer = self._plot.renderers[0]
+        renderer = self._plot.select(name="actualData")
```

Comment:
It's the same as above, and with added complication that the apps already have to implement the correct name to even start working. There is no good reason to request the default to have a special name. This makes it really complicated to use that class.